frontend/package.json

```diff
@@ -7,7 +7,7 @@
     "test:unit": "vue-cli-service test:unit",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "license-check": "license-checker --production --json --onlyAllow 'MIT;Python-2.0;Apache-2.0;BSD-2-Clause;BSD-3-Clause' --excludePackages 'company-agent-frontend@0.1.0'",
+    "license-check": "license-checker --production --json --onlyAllow 'MIT;Python-2.0;Apache-2.0;BSD-2-Clause;BSD-3-Clause;LGPL-3.0;ISC' --excludePackages 'company-agent-frontend@0.1.0'",
```

Comment:
Here we need to check if this is a compatible license. We have license alerts and a fixed list because we had to check each and every with our OSS officer, and changing something here opens a whole lot of issues on our end.

Reply:
Yes, I was a little concerned when I saw these new license requirements.  Can you check with your OSS Officer? If these licenses are not allowed, then we will have to look at other chat components (I didn't want to spend time writing a custom one). I wanted to investigate this one first, because it would fit better in the partner area as a full chat.

Reply:
The LGPL-3.0 license is due to `lamejs`. It is included as an optional dependency. We can do `npm install --no-optional` to not install optional dependencies. 

@choeppler Do you see an issue with including packages with LGPL-3.0 licenses? We'd include them as they are without modification.

Reply:
I uninstalled and re-installed `vue-advanced-chat` with the `--no-optional` flag and looks like there are no new licenses required.  Thanks for that @domwoe.

Reply:
@domwoe, as long as the LGPL-3.0-licensed packages are dynamically linked I don't see any license compatibility issues. However, to fulfil the license's obligations we'd have to provide the complete "corresponding source code" when distributing the BPA. That's more of an administrative / automation issue, though, that would need to be solved, which we avoided for the initial release...

Reply:
Thanks a lot for the clarification @choeppler.

Reply:
Looks like lamejs is required for local development debugging (running uncompiled code), so I needed to put the license back in place.